src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -717,7 +722,7 @@ mod tests {
 
     use devicemapper::{Bytes, SECTOR_SIZE};
 
-    use super::super::super::metadata::MIN_MDA_SECTORS;
+    use super::super::super::physical::MIN_MDA_SECTORS;
```

Comment:
I'm liking "backstore" for what we call the thing the flex devs sit upon. If that sounds good to you too, can we replace all instances of "physical" with "backstore", including renaming the directory?

Reply:
Done.